tests/test_loaders.py

```diff
@@ -466,3 +466,46 @@ def test_multitracks():
         assert jam.validate(), "Jams validation failed for {}.MultiTrack({})".format(
             dataset_name, mtrack_id
         )
+
+
+def test_random_splits():
+    for dataset_name in DATASETS:
+        dataset = mirdata.initialize(
+            dataset_name, os.path.join(TEST_DATA_HOME, dataset_name), version="test"
+        )
+
+        list_sum_up_1 = [
+            [0.8, 0.1, 0.1],
+            [0.7, 0.2, 0.1],
+            [0.9, 0.1, 0.0],
```

Comment:
Maybe you can add a couple of examples with different lengths to see if it does what you expect?

Reply:
Basically you could add a small test checking that you receive the number of splits that you expect

Reply:
Done! 👍